pkgs/top-level/perl-packages.nix

```
@@ -5018,6 +5018,10 @@ with self; {
       url = "mirror://cpan/authors/id/D/DP/DPARIS/Crypt-DES-2.07.tar.gz";
       hash = "sha256-LbHrtYN7TLIAUcDuW3M7RFPjE33wqSMGA0yGdiHt1+c=";
     };
+    patches = [ 
```

```suggestion
    patches = [
```

Trailing whitespace, breaks EditorConfig test

---

oop, missed that, sorry
should really get around to setting up editorconfig in helix...